restapi/stories.go

```diff
@@ -377,6 +377,20 @@ func (api api) CreateStories(w http.ResponseWriter, r *http.Request) render.Rend
 	return rest.MsgStatusOK("Stories added successfully")
 }
 
+func (api api) PublishStories(w http.ResponseWriter, r *http.Request) render.Renderer {
```

Comment:
Can we add a test for this, particularly test these two cases:

1. Changing a story that was not visible to visible
2. Changing a story that was visible to not visible